vnext/Microsoft.ReactNative.Managed.IntegrationTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Microsoft.ReactNative.Managed.IntegrationTests")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Microsoft.ReactNative.Managed.IntegrationTests")]
+[assembly: AssemblyCopyright("Copyright ©  2022")]
```

Comment:
>2022 [](http://example.com/codeflow?start=43&length=4)

Should use standardized copyright

Reply:
Yes, I agree.

Reply:
It seems that all other C# AssemblyInfo.cs in our repo are the same. We should address this issue in a separate PR.